docs/resources/aws_iam_access_keys.md

```diff
@@ -0,0 +1,164 @@
+      it { should_not exist }
+    end
+
+    # Don't let fred have access keys, using filter argument syntax
+    describe aws_iam_access_keys.where(username: 'fred') do
+      it { should_not exist }
+    end  
+
+    # Don't let fred have access keys, using filter block syntax (most flexible)
+    describe aws_iam_access_keys.where { username == 'fred' } do
+      it { should_not exist }
+    end    
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Disallow access keys created more than 90 days ago
+
+    describe aws_iam_access_keys.where { created_age > 90 } do
+      it { should_not exist }
+    end 
+
+<br>
+
+## Matchers
+
+### exists
+
+This matcher causes the control to pass if the filter returned at least one result. You can negate the meaning using should_not (that is, you expect no matches).
+
+    # Sally should have at least one access key
+    describe aws_iam_access_keys.where(username: 'sally') do
+      it { should exist }
+    end
+
+    # Don't let fred have access keys
+    describe aws_iam_access_keys.where(username: 'fred') do
+      it { should_not exist }
+    end   
+
+## Filter Criteria
+
+### active
+
+A true / false value that indicates whether the Access Key is currently Active (the normal state) in the AWS console.  `inactive` is also available.
+
+    # Check whether a particular key is enabled
+    describe aws_iam_access_keys.where { active } do
+      its('access_key_ids') { should include('AKIA1234567890ABCDEF')}
+    end
+
+### created_date
+
+A DateTime, which identifies when the Access Key was created.  See also `created_days_ago` and `created_hours_ago`.
+
+    # Don't permit creating keys on Tuesday
+    describe aws_iam_access_keys.where { created_date.tuesday? } do
+      it { should_not exist }
+    end
+
+### created_days_ago, created_hours_ago
+
+An integer, representing how old the access key is.
+
+    # Don't allow keys that are older than 90 days
+    describe aws_iam_access_keys.where { created_days_ago > 90 } do
+      it { should_not exist }
+    end
+
+### ever_used
+
+A true / false value that indicates whether the Access Key has ever been used, based on the last_used_date.  `never_used` is also available.
+
+    # Check to see if a particular key has ever been used
+    describe aws_iam_access_keys.where { ever_used } do
+      its('access_key_ids') { should include('AKIA1234567890ABCDEF')}
+    end
+
+
+### inactive
+
+A true / false value that indicates whether the Access Key has been marked Inactive in the AWS console.  `active` is also available.
+
+    # Don't leave inactive keys laying around
+    describe aws_iam_access_keys.where { inactive } do
+      it { should_not exist }
+    end
+
+### last_used_date
+
+A DateTime, which identifies when the Access Key was last used.  If the key has never been used, this will be `nil`.  See also `ever_used`, `last_used_days_ago`, `last_used_hours_ago`, and `never_used`.
```

Comment:
A DateTime identifying when the Access Key was last used.  Returns `nil` if the key has never been used. See also: `ever_used`, `last_used_days_ago`, `last_used_hours_ago`, and `never_used`.